packages/squiggle-lang/src/rescript/Utility/E/E_A.res

```diff
@@ -323,32 +323,45 @@ module Floats = {
       let continuous: array<float> = []
       let discrete = FloatFloatMap.empty()
```

Comment:
Just a thought for the future; if we later store sorted arrays, and we know we are likely to do this step later, we might want to save/cache data about which items are duplicated in these arrays, so that this algorithm could go faster.  It's probably very minor though.

Reply:
As long as we're only using this for conversion to point sets, saving the point set ensures we won't have to do it again. If we start using the continuous/discrete split for other things, then it could make sense.